rules/do-you-have-catering/rule.md

```diff
@@ -1,43 +1,34 @@
 ---
 type: rule
-archivedreason: 
-title: Do You Have Catering?
-guid: 97573f47-380c-418a-8f26-af07c1000108
-uri: do-you-have-catering
-created: 2014-12-31T00:28:41.0000000Z
+title: Catering - Do you provide catering?
```

Comment:
```suggestion
title: Do you provide event catering?
```

Reply:
@camillars please see a suggested change above! If you agree, you can confirm the change with the "Commit suggestion" button there